filetests/isa/x86/simd-arithmetic-run.clif

```diff
@@ -153,3 +153,69 @@ ebb0:
     return v8
 }
 ; run
+
+function %add_sub_f32x4() -> b1 {
+ebb0:
+    v0 = vconst.f32x4 [0x4.2 0.0 0.0 0.0]
+    v1 = vconst.f32x4 [0x1.0 0x1.0 0x1.0 0x1.0]
+
+    v2 = fadd v0, v1
+    v3 = fsub v2, v1
```

Comment:
nit: this could be optimized out as a noop in the future, so maybe we'd want to introduce the result vector and directly test the result of the add against it? And introduce a new test for sub?